app/src/main/java/com/sun/android/bai10_Recycler/WordListAdapter.kt

```diff
@@ -0,0 +1,56 @@
+package com.sun.android.bai10_Recycler
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.sun.android.databinding.WordlistItemBinding
+
+class WordListAdapter constructor(wordList: MutableList<String>) :
+    RecyclerView.Adapter<WordListAdapter.WordViewHolder>() {
+    var mWordList: MutableList<String> = mutableListOf()
+
+    init {
+        mWordList = wordList
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): WordViewHolder {
+        val binding = WordlistItemBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return WordViewHolder(binding, this)
+    }
+
+    override fun onBindViewHolder(holder: WordViewHolder, position: Int) {
+        val mCurrent = mWordList?.get(position)
+        holder.wordItemView.text = mCurrent
+    }
+
+    override fun getItemCount(): Int {
+        return mWordList?.size!!
+    }
+
+    inner class WordViewHolder(binding: WordlistItemBinding, adapter: WordListAdapter) :
+        RecyclerView.ViewHolder(binding.root), View.OnClickListener {
+        var wordItemView: TextView = binding.word
+        var mAdapter: WordListAdapter = adapter
```

Comment:
```suggestion
    inner class WordViewHolder(
    private val binding: WordlistItemBinding, 
    private val adapter: WordListAdapter
    ) : RecyclerView.ViewHolder(binding.root), View.OnClickListener {
      
```
mà sao mình lại cần adapter ở đây làm gì nhỉ

Reply:
để update Recycler view mà